src/platform/plugins/private/presentation_panel/public/panel_component/panel_header/presentation_panel_header.tsx

```diff
@@ -93,17 +73,22 @@ export const PresentationPanelHeader = <
       className={headerClasses}
       data-test-subj={`embeddablePanelHeading-${(panelTitle || '').replace(/\s/g, '')}`}
     >
-      <h2 ref={memoizedSetDragHandle} data-test-subj="dashboardPanelTitle" className={titleClasses}>
-        {ariaLabelElement}
+      <div
+        ref={memoizedSetDragHandle}
+        data-test-subj="dashboardPanelTitle"
+        className={titleClasses}
+      >
         <PresentationPanelTitle
           api={api}
+          showPanelBar={showPanelBar}
```

Comment:
I don't think we need to send this in. We return `null` above when `showPanelBar` is `false` - so there is never a situation where we reach this code and `showPanelBar` is anything other than `true`